### ISSUE TITLE
[Feature request] Functionality: Consolidate UTXOs/Cells

### ISSUE THREAD

**Issue by alejandroRbit** [Magickbase/neuron-public-issues#368]
I think that adding a functionality to consolidate Cells can be interesting from the user's point of view and also in relation to the use of the network state.

An example would be Nautilus, an ERGO wallet

![image](https://github.com/Magickbase/neuron-public-issues/assets/94946624/a08cbfe3-148e-4f25-825d-5f21ed5bc092)

REF: https://www.reddit.com/r/ergonauts/comments/14f39z9/nautilus_wallet/
https://docs.ergoplatform.com/mining/rent/

**Comment by Keith-CY** [comment]
Hi @alejandroRbit 

This function could be supported by the new feature `cell management` with PR https://github.com/nervosnetwork/neuron/pull/2859

In the cell management, cells can be selected and spent together. Once the receive address is one of the active wallet, all selected cells would be consolidted into a new one.

This feature will be delivered in the next release.

But it would be more intuitive if there's a specific button for `consolidate`, we may add it in the future cc @Danie0918

**Comment by alejandroRbit** [comment]
I agree with adding the button. The average user needs everything to be in ELI5 mode. Thank you, you guys are fantastic, keep up the good work.

**Comment by Kirl70** [comment]
Design Draft(2-15--2-16):
https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=37459%3A50762&mode=design&t=lyyUkJS0rwgDOFOY-1
<img width="968" alt="image" src="https://github.com/Magickbase/neuron-public-issues/assets/106060898/d774faac-3849-4897-83c9-aafeff208b18">

**Comment by devchenyan** [comment]
https://github.com/nervosnetwork/neuron/pull/3167
